.github/workflows/acceptance.yml

```diff
@@ -375,3 +375,57 @@ jobs:
             github-token: ${{ secrets.GITHUB_TOKEN }}
             gcs-bucket: 'sentry-visual-snapshots'
             gcp-service-account-key: ${{ secrets.SNAPSHOT_GOOGLE_SERVICE_ACCOUNT_KEY }}
+
+    openapi-diff:
+      if: ${{ github.ref != 'refs/heads/master' }}
+      runs-on: ubuntu-latest
+
+      steps:
+        - name: Checkout getsentry/sentry
+          uses: actions/checkout@v2
+          with:
+            path: sentry
+
+        - uses: dorny/paths-filter@v2
```

Comment:
Let's fork this repo as tags/branches are mutable (alternatively, use a SHA)

Reply:
@billyvg if the preference is to create a new file, then we can customize the `on` trigger. So we won't need this action.

Reply:
@NisanthanNanthakumar Will you want to block PR's from merging if there is a diff? If so, you'll want to keep this action as we will need to run it for all PRs. It's due to how status checks work - if you make them required then it has to be present for ALL PRs.

Reply:
@billyvg nah for now it's not required. It's more for reminding that changes were made and to confirm that these were the right changes

Reply:
If this is the case we prob don't need to run it on master pushes

Reply:
👍 I'll remove that trigger